### PR TITLE
Setup person, student and teacher class

### PR DESCRIPTION
**Pull request by Mikelobi**
### In this branch I:

- [ ] checked git-flow.
- [ ] Installs Linters.
- [ ] Creates class Person, Student, Teacher.
- [ ] Check linters and correct errors.

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+# Teacher class
+class Teacher < Person
+  def initialize(age:, name: 'unknown', parent_permission: true)
```
Comment (sja-thedude) [inline teacher.rb]: - [ ] Kindly add `specialization` also to the initialize constructor as per the requirements.

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+# student class
+class Student < Person
+  def initialize(age:, name: 'unknown', parent_permission: true)
```
Comment (sja-thedude) [inline student.rb]: - [ ] Kindly add `classroom` to the initialize constructor as per the requirements.

File: README.md [hunk 3]
```diff
@@ -1,73 +1,36 @@
+# ruby-my-enumerable
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly update your project heading.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Mikelobi, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Linters Passing
✔️ Professional readme

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @Mikelobi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written readme file 👍 
* Nice looking PR description 👍 
* Well written ruby classes and objects 💯 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**